akka-docs/rst/scala/stream/stages-overview.rst

```diff
@@ -84,6 +84,23 @@ If the future fails the stream is failed with that exception.
 
 **completes** after the future has completed
 
+fromFutureSource
```

Comment:
@patriknw Doc updated at the Scala side; If ok the Java one could be updated in the same way.